.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Node CI
+
+on:
+  push:
+    branches-ignore:
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-branches-and-tags